src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -426,61 +394,65 @@ const formConfig = {
                 privateRecordReleases: {
                   'ui:options': {
                     itemName: 'Private Medical Record Release',
-                    viewField: releaseView
+                    viewField: treatmentView
                   },
                   items: {
-                    privateRecordRelease: {
+                    'ui:order': [
+                      'treatmentCenterName',
+                      'privateMedicalRecordsReleaseRestricted',
+                      'view:releaseRestrictedNotice',
+                      'treatmentDateRange',
+                      'treatmentCenterAddress'
+                    ],
+                    treatmentCenterName: {
+                      'ui:title': 'Name of private provider or hospital'
+                    },
+                    treatmentDateRange: dateRangeUI(
+                      'Approximate date of first treatment',
+                      'Approximate date of last treatment',
+                      'Date of last treatment must be after date of first treatment'
+                    ),
+                    privateMedicalRecordsReleaseRestricted: {
+                      'ui:title': 'I give my consent, or permission, to my doctor to only release records related to this condition'
+                    },
+                    'view:releaseRestrictedNotice': {
```

Comment:
I just wrote a note to remind myself to write a ticket to do something with this... (It'll have to go back to the api to be put into the 4142 or 4142a, I think, but it's fine as a `view:` field for now.)